### ISSUE TITLE
Disable longer recording at the end?

### ISSUE THREAD

**Issue by jonnylander** [Bleuzen/SpotRec#10]
Hi,
I just tested SpotRec with a few songs and everything works fine. Thanks for that!

It's more a question but an issue:
Is it possible to disable the function that SpotRec records a little longer at the end?
If I'm recording a whole album, I sometimes have the problem, that the first chords of the song are in the flac-file of the previous song. 

Best regards
J.

**Comment by Bleuzen** [comment]
Hi, this is kind of expected since in many cases if not recording longer at the end the end of the song is cut of a little.
I implemented this because It is often faster to cut the end in post edit instead of recording the song again.

Since we can't get a very accurate signal from Spotify when the song ends, it is a little variable (per song/machine/internet connection I guess) so I set it to record two seconds longer at the end, just to make sure everything gets in.

If this is too long for you, feel free to edit the script before executing. The time is set here:
https://github.com/Bleuzen/SpotRec/blob/15c2e71533cb3761266a47bdb5c4ef4c1bf10358/spotrec.py#L48

Just change it to 1.0 for example (for one second) or 0 (but has the risk that songs are cut of at the end).
